express-hello-world/index.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express');
+var app = express();
+
+app.use(function (req, res, next) {
+  console.log(req.method);
+  res.send('hello hello');
+  next();
+})
```

Comment:
Usually, by the time the response has been sent, the server doesn't have any more work to do. To prevent _other_ middleware and routes from running, avoid calling `next()` after a response has been sent.